test/clients/openshift/client.go

```diff
@@ -0,0 +1,64 @@
+package openshift
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"github.com/sirupsen/logrus"
+	"k8s.io/client-go/discovery"
+	appsv1client "k8s.io/client-go/kubernetes/typed/apps/v1"
+	authorizationv1client "k8s.io/client-go/kubernetes/typed/authorization/v1"
+	batchv1client "k8s.io/client-go/kubernetes/typed/batch/v1"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+	policyv1beta1client "k8s.io/client-go/kubernetes/typed/policy/v1beta1"
+	rbacv1client "k8s.io/client-go/kubernetes/typed/rbac/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	kapi "k8s.io/client-go/tools/clientcmd/api"
+	"k8s.io/client-go/tools/clientcmd/api/latest"
+	v1 "k8s.io/client-go/tools/clientcmd/api/v1"
+)
+
+type Client struct {
+	config          *rest.Config
+	Discovery       discovery.DiscoveryInterface
```

Comment:
The kubernetes Clientset (k8s.io/client-go/kubernetes/clientset.go) is a single thing to add which includes all of these anyway so let's use that?
Later on presumably we'll be adding additional openshift clients as well.
Can this Client struct be boiled into the ClientSet? (I think it's OK to put all the Azure clients, the k8s and openshift clientsets in one struct)